### ISSUE TITLE
Temperature readings should support long term statistics

### ISSUE THREAD

**Issue by alexpomsft** [gjohansson-ST/sector#34]
Temperature readings from Sector can and should support the new statistics Home Assistant features. See https://developers.home-assistant.io/docs/core/entity/sensor/#long-term-statistics for more details.

**Comment by alexpomsft** [comment]
I took a stab at this by manually modifying my local copy of custom_components/sector/sensor.py to include:

```python
    def __init__(self, hub, coordinator, sensor, name):
        self._hub = hub
        super().__init__(coordinator)
        self._serial = sensor
        self._name = name
        self._state = None
        self._uom = TEMP_CELSIUS
        self._deviceclass = DEVICE_CLASS_TEMPERATURE
        self._stateclass = "measurement"

   ...

    @property
    def state_class(self):
        return self._stateclass
```
However, even after multiple integration reloads and HA core restarts I'm not seeing the temperature sensors available from the Statistics card. I have been able to prove that my modifications are being used, e.g. by adding "debug attributes" which show up in the UI. Not sure where to go from here...

**Comment by gjohansson-ST** [comment]
Just glancing this new "function" what you did should fix it but I also saw it could take a few hours before you have any data to display but since I have not tested myself I can not confirm.
I will work out a new beta shortly and let you test it so you can confirm if it works or not by then. Let's work it out together.

**Comment by alexpomsft** [comment]
It's been running since I posted this and the sensors still aren't showing up in the statistics card, so I don't think a few hours and another reload/restart cycle helped. Happy to test a new version!

**Comment by gjohansson-ST** [comment]
The new v0.3.4-beta1 release just made should (I hope) solve this issue.

**Comment by alexpomsft** [comment]
Can you properly tag that release so that it shows up in HACS? 
![image](https://user-images.githubusercontent.com/6499675/128769719-bbbf2aaf-77e8-4012-b319-c3ad0160fe57.png)

**Comment by gjohansson-ST** [comment]
> Can you properly tag that release so that it shows up in HACS? 
> 

It is. Turn off/on beta and it should be there.

**Comment by gjohansson-ST** [comment]
Solved in latest release v0.3.4
